### DISCUSSION TITLE
A question

### DISCUSSION THREAD

**Discussion by Spittys** [MineLittlePony/HDSkins#62] [Q&A] [answered]
can't find anything about this, and the discord took me to an MLP server so oh well, I'm sorry in advance. Just wanted to ask if theres anyway this could be used on multiplayer, for example if I needed to put it on the server or do both the players need it on their client side only, thanks

**Comment by mattmess1221** [comment, marked as answer]
This is purely a client side mod. Server-side is not required. Any other players who also have the mod will see your skin and you will see theirs.

**Comment by Spittys** [comment]
I see. Thanks for your time I appreciate it
